cmd/root.go

```diff
@@ -97,17 +97,14 @@ func Execute(cmd *cobra.Command, args []string) {
 	}
 
 	// Make sure it has a commit label before pulling
-	ostree_csum, ok := imagedata.Labels["io.openshift.os-commit"]
+	ostree_csum, ok := imagedata.Labels["com.coreos.ostree-commit"]
```

Comment:
Replacing  `io.openshift.os-commit` with `com.coreos.ostree-commit`? Just verifying as the commit message notes dropping support rather than switching.

Reply:
`com.coreos.ostree-commit` was already supported as a fallback, this is just dropping support for the old one.